string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,11 @@
+/* exported capitalize */
+const capitalize = word => {
+  let str = word[0].toUpperCase();
+  word = word.toLowerCase();
+  if (word.length > 1) {
+    for (let i = 1; i < word.length; i++) {
+      str += word[i].toLowerCase();
+    }
+  }
+  return str;
+};
```

Comment:
We can also accomplish this without a loop! Here is our solution
```
function capitalize(word) {
  return word[0].toUpperCase() + word.slice(1).toLowerCase();
}
```

Reply:
OHHHHH NICE! That's good